totest-manager.py

```diff
@@ -575,6 +582,35 @@ def totest(self):
             self.update_totest(new_snapshot)
             self.write_version_to_dashboard("totest", new_snapshot)
 
+    def send_amqp_event(self, current_snapshot, current_result):
+        if not self.amqp_url:
+            logger.info('No ttm_amqp_url configured in oscrc - skipping amqp event emission')
+            return
+
+        logger.info('Sending AMQP message')
```

Comment:
I consider this too verbose for "info" and not adding much value. How about a `logger.debug` with at least the msg_topic reflected? So one line further below